app/views/membership_applications/_membership_applications_list.html.haml

```diff
@@ -21,8 +23,8 @@
       - @membership_applications.each do |membership_application|
         %tr.applicant
           %td.membership_number= membership_application.membership_number
-          %td.name= link_to "#{membership_application.user.last_name + ', ' + membership_application.user.first_name}", membership_application_path(membership_application.id)
-          %td.company_number= membership_application.company_number
+          %td.name= link_to "#{membership_application.user.last_name + ', ' + membership_application.user.first_name}", user_path(membership_application.user)
```

Comment:
This link_to name could be shortened to:
```
link_to "#{membership_application.user.full_name}"
```

Reply:
That would lead to more changes because `full_name` is defined as `"#{first_name} #{last_name}"`, and that would mean we have to change the sorting. At the moment we sort on `last_name` instead of `first_name`.

I'd rather leave it and merge it as is.

Reply:
Yes, you're right.  Perhaps we need another User (or helper) method to return `last, first`, but thats for later.